bigquery_etl/cli/backfill.py

```diff
@@ -343,35 +343,43 @@ def scheduled(ctx, qualified_table_name, sql_dir, project_id, status, json_path=
     ConfigLoader.get("default", "project", fallback="moz-fx-data-shared-prod")
 )
 @click.pass_context
-def process(ctx, qualified_table_name, sql_dir, project_id):
-    """Process backfill entry with drafting status in backfill.yaml file(s)."""
-    click.echo("Backfill processing initiated....")
+def initiate(ctx, qualified_table_name, sql_dir, project_id):
+    """Process backfill entry with initiate status in backfill.yaml file(s)."""
+    click.echo("Backfill processing (initiate) started....")
 
     backfills_to_process_dict = get_scheduled_backfills(
-        sql_dir, project_id, qualified_table_name
+        sql_dir, project_id, qualified_table_name, status="Initiate"
     )
 
     if backfills_to_process_dict:
-        entry_to_process = backfills_to_process_dict[qualified_table_name]
+        entry_to_initiate = backfills_to_process_dict[qualified_table_name]
 
-        click.echo(f"\nValidating backfill for {qualified_table_name} via dry run:")
-        _process_backfill(ctx, qualified_table_name, entry_to_process, dry_run=True)
+        click.echo(
+            f"\nInitiating backfill(s) for {qualified_table_name} with entry date {entry_to_initiate.entry_date} via dry run:"
+        )
+        _initiate_backfill(ctx, qualified_table_name, entry_to_initiate, dry_run=True)
 
-        click.echo(f"\nProcessing backfills for {qualified_table_name}:")
-        _process_backfill(ctx, qualified_table_name, entry_to_process)
+        click.echo(
+            f"\nInitiating backfill(s) for {qualified_table_name} with entry date {entry_to_initiate.entry_date}:"
+        )
+        _initiate_backfill(ctx, qualified_table_name, entry_to_initiate)
 
-        click.echo(f"Backfill processing completed for {qualified_table_name}.")
+        click.echo(
+            f"Processed backfill for {qualified_table_name} with entry date {entry_to_initiate.entry_date}"
+        )
+    else:
+        click.echo(f"No backfill processed for {qualified_table_name}")
 
 
-def _process_backfill(ctx, qualified_table_name, entry_to_process, dry_run=None):
+def _initiate_backfill(ctx, qualified_table_name, entry_to_initiate, dry_run=None):
```

Comment:
Some nits: since the fn is called `_initiate_backfill` we know the backfill arg is the one to be initiated so we can just call it `entry`. Also type hints!

```suggestion
def _initiate_backfill(ctx, qualified_table_name: str, entry: Backfill, dry_run: bool = None):
```